crates/orderbook/src/order_quoting.rs

```diff
@@ -822,158 +540,52 @@ impl OrderQuoting for OrderQuoter {
     }
 }
 
+impl From<&OrderQuoteRequest> for PreOrderData {
```

Comment:
This just moved to the bottom of the file, since I felt it wasn't super relevant and when quickly navigating files, it was confusing to see this as the first bit of code.